### PR TITLE
GLTFLoader: Fix usage of ImageBitmapLoader.

### LINKED ISSUE #19734: GLTFLoader ImageBitmapLoader overrides loaders for types it can't handle
##### Description of the problem

Due to the recently added usage of BitmapLoader in GLFTLoader (https://github.com/mrdoob/three.js/pull/19518), using handlers for specific file formats will be overwritten from the BitmapLoader.

In r117 to support basis textures for gltf files you would just need to add
```
var basisLoader = new BasisTextureLoader();
basisLoader.setTranscoderPath( 'js/libs/basis/' );
basisLoader.detectSupport( renderer );
THREE.DefaultLoadingManager.addHandler( /\.basis$/, basisLoader );
```

This is not possible anymore, since the resolve will be overwritten here: https://github.com/mrdoob/three.js/blob/7df06a0e4054247f87a7b5aa555ca97cd6b15662/examples/jsm/loaders/GLTFLoader.js#L2077-L2093

I'm getting the following error:
```
THREE.WebGLState: TypeError: Failed to execute 'texImage2D' on 'WebGL2RenderingContext': No function was found that matched the signature provided.
    at Object.texImage2D (three.module.js:21828)
    at uploadTexture (three.module.js:22749)
    at setTexture2D (three.module.js:22274)
    at WebGLTextures.safeSetTexture2D (three.module.js:23169)
    at SingleUniform.setValueT1 [as setValue] (three.module.js:17755)
    at Function.WebGLUniforms.upload (three.module.js:18260)
    at setProgram (three.module.js:26406)
    at WebGLRenderer.renderBufferDirect (three.module.js:25441)
    at renderObject (three.module.js:25994)
    at renderObjects (three.module.js:25966)
```

* [demo](https://rawcdn.githack.com/JohannesDeml/three.js/37a8f7fd309b15900c624252147cbbec70bcc6eb/examples/webgl_loader_gltf_basis.html)
* [code](https://github.com/JohannesDeml/three.js/blob/og-dev/examples/webgl_loader_gltf_basis.html)

##### Three.js version

- [x] Dev
- [x] r118
- [ ] r117

##### Browser

- [ ] All of them
- [x] Chrome
- [ ] Firefox
- [ ] Edge

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

### PR DESCRIPTION
**Pull request by Mugen87**
Fixed #19734.

### CODE REVIEW

**Comment by Mugen87** [comment]
@mrdoob I guess it makes sense to publish this fix as `r118.2`. The current `GLTFLoader` is not usable with DDS or Basis.

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -2013,7 +2013,7 @@ THREE.GLTFLoader = ( function () {
 
 				var onLoad = resolve;
 
-				if ( useImageBitmap ) {
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js:2016]: can the `useImageBitmap` property be removed as well?

Comment (Mugen87) [reply examples/js/loaders/GLTFLoader.js:2016]: Yes, indeed! 🎉

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
> @mrdoob I guess it makes sense to publish this fix as r118.2. The current GLTFLoader is not usable with DDS or Basis.

Done!

**Comment by mrdoob** [comment]
Oh wait, but this requires new builds... r118.3 incoming... 😅

**Comment by mrdoob** [comment]
Done again!

**Pull request merged**